src/main/java/io/in_toto/lib/App.java

```diff
@@ -3,9 +3,12 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.Map;
-
+import java.io.File;
+import java.io.IOException;
 
 import io.in_toto.keys.RSAKey;
+
+//import org.bouncycastle.crypto.params.AsymmetricKeyParameter;
```

Comment:
We don't need this commented out import.

Reply:
removed